versions/f-/fbgemm.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "e19b797fb42fad6dae0f9aca0890ddc62535800f",
+      "git-tree": "27b4707b88bb2ed527d372438c406d1a97323229",
+      "version": "0.4.1",
+      "port-version": 1
+    },
+    {
+      "git-tree": "2aa09f4d6344a0c379f6b65fae72dc8117759805",
```

Here:
```suggestion
      "git-tree": "e19b797fb42fad6dae0f9aca0890ddc62535800f",
```
i.e. the original value for `0.4.1#0`.

---

OK, I have reverted it.